app/controllers/db.js

```diff
@@ -23,6 +36,40 @@ module.exports = {
         fields: ['pin']
       }
     );
+  },
+  userAuthorization: function(id) {
+    return Brother.findById(id, {
+      include: [SysAdmin, Position]
+    }).then(function(brother) {
+      if(brother.SysAdmin) {
+        return 'sysadmin';
+      };
+      if(brother.Positions.length) {
+        var positions = lodash.uniq(lodash.map(brother.Positions, 'dataValues.AuthorizationId'));
+        return q.all(lodash.map(positions, function(positionId) {
+          console.log(positionId);
+          return Authorization.findById(positionId);
+        })).then(function(authorizations) {
+          authorizations = lodash.map(authorizations, 'dataValues');
```

Comment:
What are dataValues and level here? There don't seem to correspond to anything that I can see, help a noob out.